### PR TITLE
Feat/migaloo assets

### PR DESCRIPTION
**Pull request by 0xFable**
Related to #1658
I synced up the fork and made a new branch with the commits adding Migaloo's Tokenfactory Assets.
Also got some more svgs

### CODE REVIEW

**Comment by 0xFable** [comment]
@JeremyParish69 Could you take a look at this new PR? I fixed the merge conflicts 🤝

**Comment by JeremyParish69** [comment]
@0xFable This still is trying to add the submodules...

File: migaloo/assetlist.json [hunk 1]
```diff
@@ -22,6 +22,69 @@
...
+        {
+          "denom": "ampWHALE",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo1436kxs0w2es6xlqpp9rd35e3d0cjnw4sv8j3a7483sgks29jqwgshqdky4/ampWHALE",
+      "name": "ampWHALE",
+      "display": "ampWHALE",
+      "symbol": "ampWHALE",
+      "coingecko_id": "ampWHALE",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:42]: Please remove the coingecko_id as it doesn't exist

File: migaloo/assetlist.json [hunk 2]
```diff
@@ -22,6 +22,69 @@
...
+        {
+          "denom": "fable",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo18a9m9stu3dyvewwcq9qmp85euxqcvln5mefync/fable",
+      "name": "FABLE",
+      "display": "FABLE",
+      "symbol": "FABLE",
+      "coingecko_id": "FABLE",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:84]: Please remove the coingecko_id as it doesn't exist

File: migaloo/assetlist.json [hunk 3]
```diff
@@ -22,6 +22,69 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "fable",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo18a9m9stu3dyvewwcq9qmp85euxqcvln5mefync/fable",
+      "name": "FABLE",
+      "display": "FABLE",
```
Comment (JeremyParish69) [inline migaloo/assetlist.json:82]: the display [denom] must match one of the denom_units defined above. so this should be 'fable' (lowercase).

**Comment by JeremyParish69** [comment]
@0xFable , my new PR resolves all these issue--I'd just add the new svg images to it

**Comment by 0xFable** [comment]
Just seeing this now I will update with SVGs tonight

**Pull request closed without merging**